test/generators/ModelGenerators.scala

```diff
@@ -101,7 +101,7 @@ trait ModelGenerators extends BaseGenerators with JavaTimeGenerators {
         status   <- arbitrary[DepartureStatus]
         created  <- arbitrary[LocalDateTime]
         updated  <- arbitrary[LocalDateTime]
-        messages <- nonEmptyListOfMaxLength[MessageWithStatus](2)
+        messages <- nonEmptyListOfMaxLength[MessageWithStatus](1)
```

Comment:
Why was this value changed to 1? This differs from the Arbitrary declaration in the arrivals backend.

Reply:
because our use of it was for a departure with one message. This wasn't behaving as expected, and was causing intermittent issues as a result. As it was only used there, I changed this here.